models/import.js

```diff
@@ -1,5 +1,285 @@
+const DateString = Match.Where(function (dateAsString) {
+  check(dateAsString, String);
+  return moment(dateAsString, moment.ISO_8601).isValid();
+});
+
+class TrelloCreator {
+  constructor() {
+    // the object creation dates, indexed by Trello id (so we only parse actions once!)
+    this.createdAt = {
+      board: null,
+      cards: {},
+      lists: {},
+    };
+    // the labels we created, indexed by Trello id (to map when importing cards)
+    this.labels = {};
+    // the lists we created, indexed by Trello id (to map when importing cards)
+    this.lists = {};
+    // the comments, indexed by Trello card id (to map when importing cards)
+    this.comments = {};
+  }
+
+  checkActions(trelloActions) {
+    check(trelloActions, [Match.ObjectIncluding({
+      data: Object,
+      date: DateString,
+      type: String,
+    })]);
+    // XXX perform deeper checks based on type
+  }
+
+  checkBoard(trelloBoard) {
+    check(trelloBoard, Match.ObjectIncluding({
+      closed: Boolean,
+      labels: [Match.ObjectIncluding({
+        // XXX check versus list
+        color: String,
+        name: String,
+      })],
+      name: String,
+      prefs: Match.ObjectIncluding({
+        // XXX check versus list
+        background: String,
+        // XXX check versus list
+        permissionLevel: String,
+      }),
+    }));
+  }
+
+  checkLists(trelloLists) {
+    check(trelloLists, [Match.ObjectIncluding({
+      closed: Boolean,
+      name: String,
+    })]);
+  }
+
+  checkCards(trelloCards) {
+    check(trelloCards, [Match.ObjectIncluding({
+      closed: Boolean,
+      desc: String,
+      // XXX check idLabels
+      name: String,
+      pos: Number,
+    })]);
+  }
+
+  /**
+   * must call parseActions before calling this one
+   */
+  createBoardAndLabels(trelloBoard) {
+    const createdAt = this.createdAt.board;
+    const boardToCreate = {
+      archived: trelloBoard.closed,
+      color: this.getColor(trelloBoard.prefs.background),
+      createdAt,
+      labels: [],
+      members: [{
+        userId: Meteor.userId(),
+        isAdmin: true,
+        isActive: true,
+      }],
+      // current mapping is easy as trello and wekan use same keys: 'private' and 'public'
```

Comment:
They recently renamed the “organization” into “tersonal team”. This feature is in the free plan, you can create an organization with only one user in it (you) with the `+` icon on the header bar, and then select “Team visible” in the visibility setting.

I just tried and the `permissionLevel` is `"org"`.